src/irc/core/irc-expandos.c

```diff
@@ -138,7 +138,7 @@ static char *expando_cumode_space(SERVER_REC *server, void *item, int *free_ret)
                 return "";
 
 	ret = expando_cumode(server, item, free_ret);
-	return *ret == '\0' ? " " : ret;
```

Comment:
ret must not be freed unless free_ret is set. Probably it won't ever be set if *ret is 0